### DISCUSSION TITLE
Correct configuration for monitoring only

### DISCUSSION THREAD

**Discussion by hspindel** [nutdotnet/WinNUT-Client#149] [Q&A]
I want to monitor an APC UPS from Windows. The UPS is connected to a remote Synology.

All I want is to monitor. I don't want Windows to shut down if the UPS is at low battery. What settings do I use to stop WinNUT-Client from initiating a Windows shutdown ever?

Thank you.

**Comment by gbakeman** [comment]
Hi @hspindel ,

It looks like we don't really have an easy way to disable automatic shutdowns in WinNUT. Looking at my settings, I think you want to set the **Shutdown if battery lower than** and **Shutdown if runtime lower than** fields to 0. If you set your settings as I do in my picture below, then I believe your computer will not react to the NUT server it's connected to. You may also be interested in issue #92 where I'm in the early stages on planning a more flexible system of action handling in WinNUT. Feel free to subscribe and provide ideas/feedback that you might have.

![image](https://github.com/nutdotnet/WinNUT-Client/assets/842942/a81bba39-dc0a-48fe-bae3-4e1fc843b864)

**Reply by hspindel** [reply]
Thank you for the reply.  I had already set those fields to zero in hopes that that would help.  I really don't want to test it by pulling the plugs on the UPSes and draining the batteries. ;-)

I read issue #92, and while that sounds potentially helpful it would have to be implemented so that nothing other than the script was run and the script could be configured to do nothing.

I appreciate your work on this, and it is working well in the few days I've been using it!
